pyproject.toml

```diff
@@ -1,20 +1,21 @@
 [build-system]
-requires = ["setuptools>=64"]
-build-backend = "setuptools.build_meta"
+requires = ["hatchling", "uv-dynamic-versioning"]
+build-backend = "hatchling.build"
+
+[tool.uv-dynamic-versioning]
+pattern = "default-unprefixed"
 
 [project]
-name = "dicomweb-client"
+name = "pydicomweb-client"
```

Comment:
We called the repository pydicomweb, so I would call the package in the same way